### ISSUE TITLE
Bit(1) is read incorretly

### ISSUE THREAD

**Issue by mauricewendt** [rstats-db/RMySQL#168]
I am using RMySQL 0.10.9 and Bit(1) columns are returned as 0, but there are all 1

**Comment by karina-rebuli** [comment]
Yes, this is a known behavior. You can use SELECT  IF( field, 1, 0 ) AS field FROM table.

**Comment by emillykkejensen** [comment]
Are there any plans for fixing this in the near future?
